### ISSUE TITLE
Bug: More Tuning options result in less results

### ISSUE THREAD

**Issue by konstantingoretzki** [sullo/nikto#719]
### Expected behavior
Scanning with `-T 12` should contain the results for `-T 1` and `-T 2`.

### Actual behavior
If I combine both scan types via scan tuning I do not get any finding at all (I've got them if I only do one after another) and less request are made.

### Steps to reproduce

1. Use any system to scan. I use a local docker container of the dvwa image, so: 
`docker run --rm -it -p 80:80 vulnerables/web-dvwa`
2. Scan via nikto:
- T1: `sudo docker run --net=host --rm sullo/nikto -h http://localhost:80 -Tuning 1`
  ```
  ...
  + /login.php: Admin login page/section found.
  + 2151 requests: 0 error(s) and 6 item(s) reported on remote host
  + End Time:           2021-04-24 00:55:27 (GMT0) (3 seconds)
  ---------------------------------------------------------------------------
  + 1 host(s) tested
  ...
  ```

- T2: `sudo docker run --net=host --rm sullo/nikto -h http://localhost:80 -Tuning 2`
  ```
  ...
  + OSVDB-3268: /config/: Directory indexing found.
  + OSVDB-3268: /docs/: Directory indexing found.
  + /.gitignore: .gitignore file found. It is possible to grasp the directory structure.
  + 1238 requests: 0 error(s) and 8 item(s) reported on remote host
  + End Time:           2021-04-24 00:56:51 (GMT0) (2 seconds)
  ---------------------------------------------------------------------------
  + 1 host(s) tested
  ```
- T12: `sudo docker run --net=host --rm sullo/nikto -h http://localhost:80 -Tuning 12`
  ```
  ...
  !!no /login.php or /config/ , /docs/ and /.gitignore found!!
  + 493 requests: 0 error(s) and 5 item(s) reported on remote host
  + End Time:           2021-04-24 00:58:33 (GMT0) (2 seconds)
  ---------------------------------------------------------------------------
  + 1 host(s) tested
  ```
3. Compare all scan results.

### Nikto version
 I've used nikto via the self-build docker version and also the preinstalled version on Kali Linux.
Both are `Nikto v2.1.6`. Same result with both installations.

Last commit of the git version:
```
commit d27173173bebac29fa89ef3a493febab16594a12 (HEAD -> master, origin/master, origin/HEAD)
Author: sullo <sullo@cirt.net>
Date:   Thu Apr 8 16:13:22 2021 -0400

    Uncomment report-to so it's not reported #717
```

### Further technical info
I've compared some outputs via `-D D`.
Not sure if I can naively can compare these outputs, but for me it's noticable that there are scans / results missing compared to doing the scans sequently.
```
nikto -h http://localhost:80 -Tuning 12 -D D > combined.txt
nikto -h http://localhost:80 -Tuning 1 -D D > one.txt    
nikto -h http://localhost:80 -Tuning 2 -D D > two.txt
                                                                            
wc -l combined.txt                        
> 28834 combined.txt
                                                                            
wc -l one.txt                                        
> 147035 one.txt
                                                                            
wc -l two.txt 
> 81553 two.txt

cat combined.txt | grep .gitignore
> $? is 1 - no match
cat one.txt | grep .gitignore
> $? is 1 - no match                                                                         
cat two.txt | grep .gitignore                                   
>                'uri' => '/.gitignore',
>                'uri_requested' => '/.gitignore',
>                'uri' => '/.gitignore',
> + /.gitignore: .gitignore file found. It is possible to grasp the directory structure.
```

### Further info
Not sure if this is the same as #700 as I am not 100% sure if I have understood @drwetter correctly.

**Comment by drwetter** [comment]
Yes it is quite similar what I observed

**Comment by sullo** [comment]
This has been fixed (I believe) in the master-2.5.0 branch. Please checkout that branch and test, and reply here with confirmation it is working. 

Note that if you run something like `-T 12` the request total will **not** be the same as if you run `-T 1` and `-T 2` separately, as some tests may match both and thus duplicated in the individual flag runs.

**Comment by konstantingoretzki** [comment]
Awesome! This seem to work now.
Thank you very much.

Below are my new test results:
```
sudo docker run --net=host --rm sullo/nikto -h http://localhost:80 -Tuning 12
- Nikto v2.5.0
---------------------------------------------------------------------------
...
+ /config/: Directory indexing found.
+ /login.php: Admin login page/section found.
+ /docs/: Directory indexing found.
+ /.gitignore: .gitignore file found. It is possible to grasp the directory structure.
+ 2875 requests: 0 error(s) and 9 item(s) reported on remote host
+ End Time:           2021-05-09 14:56:15 (GMT0) (6 seconds)
---------------------------------------------------------------------------
+ 1 host(s) tested
```

Good to know that there are some tests that are in both categories.
